### DISCUSSION TITLE
Polling Query

### DISCUSSION THREAD

**Discussion by Teloriun** [openscopeproject/TrguiNG#56] [Q&A] [answered]
Hi

I've changed the polling values to 1 second for active and torrent details (see below) but I'm not sure it's working right, misunderstood setting or I've set it incorrectly?  
If a torrent is active uploading or downloading, shouldn't the rates be changing every second?  It seems to work OK in the torrent details (bottom panel), but not the main window (or the very bottom app bar).

Included how I had it set on Transgui which seems to work.

ps. any plans on having the average out transfers option?

![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/fe23adf4-57b2-4c70-8fa0-78d4f8a8bd4c)

![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/8360e7b4-731c-434a-8b61-d7f741e8fda0)

Running on Windows 10 Pro / TrguiNG_0.8.1_x64-setup.exe

**Comment by qu1ck** [comment]
Looks like the torrents active polling interval setting only takes effect after the app is restarted or window is hidden/shown.
That's a bug but you can just toggle the window once for now as a workaround.

**Reply by qu1ck** [reply, marked as answer]
This is fixed now, will be in next version or you can compile yourself.

**Reply by Teloriun** [reply]
Thanks for the prompt response and fix!
